Gemfile

```diff
@@ -42,7 +42,8 @@ gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
 gem 'staccato'
 gem 'twilio-ruby'
-gem 'two_factor_authentication', github: 'Houdini/two_factor_authentication'
+gem 'two_factor_authentication', github: 'amoose/two_factor_authentication',
+                                 branch: 'feature/more_options'
```

Comment:
@sbc100 I am documenting the upstream PR work in an issue: https://github.com/18F/identity-private/issues/708